docs/pages/admin-guides/management/security/client-timeout.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Securing Sessions with Client Timeout Enforcement
+description: How to implement idle client timeouts.
+---
+
+The `client_idle_timeout` in Teleport is a configurable setting that helps improve security by terminating inactive sessions after a specified period. It can be applied globally or per role, 
+allowing for flexibility based on your organization's security policies. The `client_idle_timeout` configuration ensures that SSH sessions, desktop sessions, kubectl exec or database 
+connections that remain inactive for a certain period of time are automatically terminated. 
+
+You can also use it to set limits for SSH sessions initiated *from* the web interface (note that this is distinct from `web_idle_timeout`, which sets idle timeout for the Web UI). 
+This helps to mitigate risks associated with unattended sessions, such as unauthorized access.
+
+## Use cases
+
+- Security compliance: Many organizations require idle timeout enforcement as part of their security policies, ensuring that inactive sessions are not left open.
+- Mitigating risk: If users forget to disconnect from a session, an idle timeout ensures that they are logged out automatically after a set period of inactivity, reducing the risk of unauthorized access.
```

Comment:
```suggestion
- Risk mitigation: If users forget to disconnect from a session, an idle timeout ensures that they are logged out automatically after a set period of inactivity, reducing the risk of unauthorized access.
```

Since the previous use case was listed as a noun